nmrfx-processor/src/test/java/org/nmrfx/processor/datasets/ProcessTest.java

```diff
@@ -1,18 +1,48 @@
 package org.nmrfx.processor.datasets;
 
 import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.ClassRule;
 import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
 import org.python.util.PythonInterpreter;
 
 import java.io.File;
+import java.io.FileNotFoundException;
 import java.io.IOException;
+import java.nio.file.FileSystems;
 import java.nio.file.Path;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assume.assumeFalse;
+
 public class ProcessTest {
+
+    private static final String VALID_SUBMODULE_LOCATION = "nmrfx-test-data/valid";
+    private static final String FID_SUBMODULE_LOCATION = "nmrfx-test-data/testfids/";
+    private static final String ERR_MSG = "File doesn't exist: ";
     public static String scriptHome = "src/test/resources/process_scripts";
-    public static String fidHome = "../../nmrfxp_tests/testfids/";
-    public static String tmpHome = "../../nmrfxp_tests/tmp/";
-    public static String validHome = "../../nmrfxp_tests/valid";
+    private static String fidHome;
+    private static String tmpHome;
+    private static String validHome;
+
+    @ClassRule
+    public static final TemporaryFolder tmpFolder = TemporaryFolder.builder()
+            .parentFolder(new File(System.getProperty("user.dir")))
+            .assureDeletion()
+            .build();
+
+    @BeforeClass
+    public static void setup() {
+        Path parent = FileSystems.getDefault()
+                .getPath("")
+                .toAbsolutePath()
+                .getParent();
+        // Python scripts expect path with / not \, so replace if present
+        fidHome = parent.resolve(FID_SUBMODULE_LOCATION).toString().replace("\\", "/") + "/";
+        validHome = parent.resolve(VALID_SUBMODULE_LOCATION).toString().replace("\\", "/") +"/";
+        tmpHome = tmpFolder.getRoot().toString().replace("\\", "/") + "/";
```

Comment:
Was trying to avoid having to use `../../` in a hardcoded string and have the path automatically created, but had issues with the python scripts expected file separators. Not sure if this is any better than the hardcoded strings.